docs/source/pip-comparison.rst

```diff
@@ -0,0 +1,26 @@
+===================
+Conda for pip users
```

Comment:
I have no idea what "Rosetta Stone" means. I discovered the section very late and had no idea what to expect under that title. Not sure if this is an international thing, otherwise I would suggest to use as plain English as it gets.